docs/diagrams/SecureDrop_DataFlow.draw

```diff
@@ -1 +1 @@
++WT/r0PsIxfRSfpOtdx3tby+iH5nzcnf4yESNPTiC3/rve6vp39wOfy9y3/d/8RfE1H9o4WNbyvbV+nQLx72BwE4Qd17mXcp0tcCnEDvxu/cp/jeDPBfUYz4KxTjb5rpwANyGMb8PNOBOaUM/m4zHVb+8PdMh3l7msYD7y7+Hk9d3SZP3MJIdN650t45nbY7Zz4EHVy7ZCKiHLQ+MkK8OnVtPQACKpAdZeKUuF6NSYk1hmLyVhqsQGOcc7gnzfR6aFx+jPLL1q39fFSI/jJ656Kcel/zfS8ro6wP5uMVp1BBoOlp7q4meJXe6tcxdzz3VAB4c+g1y9rCz/xttKszSjpLDPVF4px0vekvCr/2ZGE79IAWiB5MQkwOnkrnoirgWrKGoPMmigALXKuz0o3e+cjin3iY4hRP6MGqbyMBkShSxPnkQSCO7Ma6w9bpt97EEfYriuYZOU4OpmgM++p63iTLsl0KaDNC6UaaTXqe0VMCF9da1BC5XXBw3AJ23o7h9rCwn9Zdf9bsY96y3J5LmspnLn42lKsQm899dTHb2oZmdT6L9VSvpC1E5d5xItfTDB/yDWIBMusI4z1O6ImoUHFyD9jQ1JwCRAzAt48FRm3ZjdDYdLi6sFhR95tqnlYKUzGUejBzxRZqf8rCjdRxx3Lcn6vhOofJOd2u09VpJVgh7CkEvMOj9SyZMWxfXUTfQumWHA87jNnBoXBEK/cQnJmLyJW4plRYyzgdJ078XpOP4cqu7U6MmZPGKRcpD4hDRFDUiisNbK+fUXWMzV6NZhToLZ60piKjo1CBJsCebbFkCh2EvhFOzSxgaqBohjxgWMKkepwUTBIlh62yOtEI4GOTs+GEjyghhBi12ZXmapaAqIcTcpQp5gII35kz2Ww2NIe0JLJt57RoJ7hnB2HVsJG+cYfZdgcScK75o4fi9HKPtI5QS+2dowV1gFduHJmTsYtOiODpY1UWdUbY5C46hAKeYMdLC+jYR1Y/nAA9+DFemyqJgJms2RI1bXiauzYSl+j7WG/qiCuhGlAiT6ujKgozCeFrQCmFqtq10i4X1zItLja6PodIIRt5ZaotnwUk/vwMqHJjGiJDdNxvm0CeAXo3F2U45TtaYhXo5JccICpsabLfMoTYx7Qs5XoqRFnmUBvAIxkqWysW3e1IZsQ53wvLR3qKYuW6gBPZvdAMuq83UHb1k/bSXNdkxBy2Tcpk9mJWG/QUs3rsZCZKlXImMCiGzp4/wxZ79bAFoRRiXlsTQvxx7bF5OFSupe26GxHjqicSsz+n9O5ghmp0OFAsPO7D6LLOQS2asScHjQ5uCFu7iFQUzxjWZ7Vel+tpjaqwnJdbf3fqUhPJ5nA+lXVIFHGk+jGQuBj7sgFRgJkOH0lBRXVFkU3WT9u4sU8Gycpd6LSADTI/H3mYwcsRDKnWdoFyjeGa1MNJEaMNxU2nyMjDmQcTM21zew6vRmK15Xrb51x0upSr0uZXVVDPQq1pyIA7POqSUeHsyq3sXmnV3eRFABj8CuGEOTFFr7fibRT3gRYNbaxcqVPsDKcmUabIysGQXJa4km3TgRud6wxJ6iE5c2quStgmV5GL7vrQ5agB0usKmrWKkyblEpQDeEQUf9Jk+tVPO1fs/bOjKHvxoOcEtS6CyOHAKHcFO3mLlrnQkZkCWj7kQl2dc3StEpew2ZMuEXvvQNBJgDqFa+wVYi/ro3mW++U9iUAgRtW8uHq0NQwvZnhfCLndcTBThjgnQ0EbYhjNnuq1GyNLCdTcDCEkyfBp1uA65i/zmaOSs2REQPaBQa1WOM2S9KbjW2P5ZIcNLeCcHpKwP1ekXqARXcKaW89MG6Q13wC8ji2gA90xSXYFhj4/q+1BOsjHakAY/0KMcC4BM7yYcP5iekmGVNkFs6FLD2S2S01Fmc2+cwDTK+2n+gy45EDd7+Svpm2ol2dCnyK/X9uWyk8c4BvdqVM5tJtAtocaz+FzdHIYGryvoqfuquU4W8+01dbKSFOtkIRtiNGNfWKFIb0UogacAW1szGcrg2aF7Yg5hmNqV2uKcoNGcDbPHVg7aTwao5NdK9TOsuZU4VjA2MOBaM/8ZChXMkGUkfRiz9viFXFK1nKj0ShlORVUx2rjxOsNfBYZhwiqMw97ob0Ch0x5WfTkaTX3abmrNLazUJZk0DXpTrGVeS2wDuEZCLIyVDAYRHdhoj1hBZ2S663NMo4TBTXpHjNkW8vExgBiBPNBUKi66vKOzFka4qT6so24Q8/Z2X5Bx3IbmVKBYLM80KQ1kPVRz+Kj6KIdch1GptDyLXGMy3nZJGTkRSPsDolPA25F5ZLCjgVocIGiDarJX7V5RTeSqh4XJ8TGFWlP6H47O4WIrTdGBSkiYE0egdtitK4nXA9BRVd4uTmrZYEMlzQ7QPNlWsFNo7raeasc0+MudoemZeA98EISG+0dLcQTpuZdThUafadwIi4S2uF8I53n97Li+7PgaPwqB332vCAE2UrYwaMArW4/Nsf9IYNKbtWnN73TCVtoQiu9pEuZBA/hp5VbzfGUUpHQRB0T0PVOQ2uHtC9+EwE2Sc8m5yy77ugCs91AFk8evaqKSlYuJecXJ2QLk9S4p3HmWvPe+dKO3nWDzGsNpHl5f8HGPoQE/4juO+5mB5urte4UVt5wb/m7/FNm8EZpP26vdoTH3+QyfO9mAeqp9/XLAfAXyPd+U/kZoT5A+fm1qZKXY0zk1UEm/NT79UGCTAp6MWvw5hjzS4PVN+7zR0NM6gMUf1/ECQGRbwXbj4/iv4iyz1vqIyfwKBT9zhH6+3O8rwbi3eHr+/v8USAib+zy/wdyZq86mQUYlZ7jeHF9fg2exCM8P1r6jaTvCvr3tu71qvH+QNHvyS/fq+AnKb8fsckT58e7HFH5drvLHz28coe2F7D7CMB3Pv6H/TRN3/2N7vu+3q1d6+45n7sTX/1eX1//ViQuP3Y1qKb/fXnnNcmuDkJwxf8D</diagram></mxfile>
```

Comment:
I think the Export device should be outside of the airgapped area to make it clear to a user that the devices transfers across environments.

Reply:
Done in 82c9436ccab0d070d09d1b86a183c0102232891a (also moved the "printed documents" icon for the same reason).